src/providers/ldap/sdap_id_op.c

```diff
@@ -592,6 +592,14 @@ static void sdap_id_op_connect_done(struct tevent_req *subreq)
             }
         }
         ret = sdap_id_conn_data_set_expire_timer(conn_data);
```

Comment:
I agree, the failure to set a timer doesn't affect this connection, but only its 'renewal', which is no reason to bring the back end offline.